elytron/WFLY-19269-OCSP-stapling-support.adoc

```diff
@@ -0,0 +1,199 @@
+---
+categories:
+# Add any category for this proposal
+# if missing, add it to _data/widfly-categories and use its id
+---
+= [Experimental|Preview|Community|default]Template <INSERT TITLE HERE>
+:author:            Prarthona Paul
+:email:             prpaul@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Online Certificate Status Protocol (OCSP) is one of the methods of checking whether or not a certificate presented by a client is valid or not. It is a protocol for determining the status of a certificate and is described in http://www.ietf.org/rfc/rfc2560.txt[RFC 2560]. 
```

Comment:
s/certificate presented by a client is valid or not/certificate is valid or not

Reply:
Fixed!